### ISSUE TITLE
It says I'm on Windows :(

### ISSUE THREAD

**Issue by AlekSi** [jeremyevans/home_run#1]
```
$ uname -a
Darwin Nuro4ek.local 10.4.0 Darwin Kernel Version 10.4.0: Fri Apr 23 18:28:53 PDT 2010; root:xnu-1504.7.4~1/RELEASE_I386 i386

$ ruby --version
ruby 1.8.7 (2010-04-19 patchlevel 253) [i686-darwin10.4.0], MBARI 0x6770, Ruby Enterprise Edition 2010.02

$ rake mem_bench
(in /Users/AlekSi/.rvm/gems/ree-1.8.7-2010.02/gems/home_run-0.9.0)
Memory benchmarks not supported on Windows
```

**Comment by labria** [comment]
It checks:

RUBY_PLATFORM =~ /win|w32/

and Dar_win_ matches =)

**Comment by jeremyevans** [comment]
Better guessing about Windows using RUBY_PLATFORM, closed by e220c19b91f9ebdf967d906704e5f0b2b2929ce2

**Comment by jeremyevans** [comment]
Fixed: http://github.com/jeremyevans/home_run/commit/e220c19b91f9ebdf967d906704e5f0b2b2929ce2

mswin|mingw should work for all Windows versions, without any false positives, hopefully.
